client/src/components/RhythmsWithSlider.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import DualRangeSlider from "./DualRangeSlider";
+
+const RhythmsWithSlider = ({ rhythms }) => {
+	const [selectedRange, setSelectedRange] = useState([1400, 2023]);
```

Comment:
Better to use something like Date.now() and get the year, but good illustration of the max/min